test/ComposableCoW.stoploss.t.sol

```diff
@@ -269,6 +272,46 @@ contract ComposableCoWStopLossTest is BaseComposableCoWTest {
         stopLoss.getTradeableOrder(safe, address(0), bytes32(0), abi.encode(data), bytes(""));
     }
 
+    function test_OracleRevertOnExperiedOrder_fuzz(
+        uint32 currentTime,
+        uint32 validTo
+    ) public {
+        // enforce expired order
+        vm.assume(currentTime > validTo);
+
+        vm.warp(currentTime);
+
+        StopLoss.Data memory data = StopLoss.Data({
+            sellToken: mockToken(SELL_TOKEN, DEFAULT_DECIMALS),
+            buyToken: mockToken(BUY_TOKEN, DEFAULT_DECIMALS),
+            sellTokenPriceOracle: mockOracle(SELL_ORACLE, 100 ether, block.timestamp, DEFAULT_DECIMALS),
+            buyTokenPriceOracle: mockOracle(BUY_ORACLE, 100 ether, block.timestamp, DEFAULT_DECIMALS),
+            strike: 1,
+            sellAmount: 1 ether,
+            buyAmount: 1 ether,
+            appData: APP_DATA,
+            receiver: address(0x0),
+            isSellOrder: false,
+            isPartiallyFillable: false,
+            validTo: validTo,
+            maxTimeSinceLastOracleUpdate: 15 minutes
+        });
+
+        vm.expectRevert(
```

Comment:
It would be better if there was also a test that shows an order is valid if warping to `validTo`, all other tests use `type(uint256).max` for the valid to so the positive boundary condition isn't tested on a different number.